contract/migrations/11.0.4.0.0/post-migration.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Eficent - Miquel Raïch
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+def migrate(cr, version):
+    """All previous contract where of 'sale' type."""
+    if not version:
+        return
+    cr.execute(
+        """
+        UPDATE account_analytic_account
+        SET contract_type = 'sale'
```

Comment:
I'm thinking you don't need this putting as default value `sale` in the field `contract_type`

Reply:
But this way they are already filled by the migration to avoid any rare case.

Reply:
There's no rare case. This will simply be filled by ORM default. Which case can be?

Reply:
This is really not needed.